name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/XTBPolandPDFExtractor.java

```diff
@@ -0,0 +1,283 @@
+                                        // ACC, EUR 11:18:09
+                                        // @formatter:on
+                                        section -> section //
+                                                        .attributes("shares") //
+                                                        .match("^(\\d+) (\\d+) ([A-Z0-9\\\\._-]{1,10}(?:\\\\.[A-Z]{1,4})?)\\s+(.*?),(.*?)\\s+(?<shares>[.,'\\d]+) ([\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) \\w+ ([\\.,'\\d]+) ([\\.,'\\d]+) .* (?<currency>[A-Z]{3}) ([\\.,'\\d]+) ([\\.,'\\d]+) ([\\.,'\\d]+) ([\\.,'\\d]+).*$") //
+
+                                                        .assign((t, v) -> t.setShares(asShares(v.get("shares")))),
+
+                                        //
+                                        // @formatter:off 
+                                        // Executed Buy Orders for Fractional Rights 
+                                        //1 2004419557 FWIA.DE Invesco, UCITS, XTB 0.2636 03.09.2025 rynkowe 6.67600 1.75979 Prawa ułamkowe EUR 1.0000 0.00 0.00 0.00
+                                        //ACC, EUR 11:18:08 dot. ETF
+                                        // @formatter:on
+                                        section -> section //
+                                                        .attributes("shares") //
+                                                        .match("^(\\d+) (\\d+) ([A-Z0-9\\\\._-]{1,10}(?:\\\\.[A-Z]{1,4})?) (.*?) (?<shares>[\\.,'\\d]+) ([\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) \\w+ ([\\.,'\\d]+) Prawa \\w+ (?<currency>[A-Z]{3}) ([\\.,'\\d]+) ([\\.,'\\d]+) ([\\.,'\\d]+) ([\\.,'\\d]+).*$") //
+
+                                                        .assign((t, v) -> t.setShares(asShares(v.get("shares"))))
+                                        )
+
+                        
+                        .oneOf(
+                                        //
+                                        // @formatter:off
+                                        // Executed Buy Orders for Financial Instruments (OMI) 
+                                        // 1 2004419557 FWIA.DE Invesco, UCITS, BATE 30.0000 03.09.2025 rynkowe 6.67500 200.25000 ETFs EUR 1.0000 0.00 0.00 0.00 
+                                        // ACC, EUR 11:18:09
+                                        // @formatter:on
+                                        section -> section //
+                                                        .attributes("date", "time") //
+                                                        .match("^(\\d+) (\\d+) ([A-Z0-9\\\\._-]{1,10}(?:\\\\.[A-Z]{1,4})?)\\s+(.*?),(.*?)\\s+([.,'\\d]+) (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) \\w+ ([\\.,'\\d]+) ([\\.,'\\d]+) .* ([A-Z]{3}) ([\\.,'\\d]+) ([\\.,'\\d]+) ([\\.,'\\d]+) ([\\.,'\\d]+).*$") //
+                                                        .match("^([A-Z]{3})\\, ([A-Z]{3}) (?<time>[\\d]{2}:[\\d]{2}:[\\d]{2}).*")
```

Comment:
```suggestion
                                                        .match("^[A-Z]{3}, [A-Z]{3} (?<time>[\\d]{2}:[\\d]{2}:[\\d]{2}).*$")
```

In general... don't create RegEx groups where they are not needed.
The comma does not necessarily have to be invalidated.